repos/coreclr.proj

```diff
@@ -46,5 +46,23 @@
     <EnvironmentVariables Include="OfficialBuildId=$(OfficialBuildId)" />
   </ItemGroup>
 
+  <Target Name="CopyTools"
+          AfterTargets="Package">
+    <ItemGroup>
+      <CoreClrTools Include="$(ProjectDirectory)bin/Product/Linux.x64.Release/ilasm" />
+      <CoreClrTools Include="$(ProjectDirectory)bin/Product/Linux.x64.Release/ildasm" />
+      <CoreClrTools Include="$(ProjectDirectory)bin/Product/Linux.x64.Release/libcoreclr.so" />
+      <CoreClrTools Include="$(ProjectDirectory)bin/Product/Linux.x64.Release/System.Private.CoreLib.dll" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <CoreClrDestinationDir>$(ToolPackageExtractDir)ilasm</CoreClrDestinationDir>
```

Comment:
Confusing dir name to contain `ilasm`, `ildasm`, and more.

Reply:
Changed this to `coreclr-tools`.